packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/shared.ts

```diff
@@ -10,18 +10,28 @@ export const DEFAULT_ACTION_VIEW_DISPLAY_TEXT: Omit<
   actionCancelLabel: 'Cancel',
   actionExitLabel: 'Exit',
   actionDestinationLabel: 'Destination',
+  // status display
   statusDisplayCanceledLabel: 'Canceled',
   statusDisplayCompletedLabel: 'Completed',
   statusDisplayFailedLabel: 'Failed',
   statusDisplayTotalLabel: 'Total',
   statusDisplayQueuedLabel: 'Not started',
+  // table task display
+  tableStatusDisplayCanceledLabel: 'Canceled',
+  tableStatusDisplayCompletedLabel: 'Completed',
+  tableStatusDisplayFailedLabel: 'Failed',
+  tableStatusDisplayOverwritePreventedLabel: 'Overwrite prevented',
+  tableStatusDisplayInProgressLabel: 'In progress',
+  tableStatusDisplayQueuedLabel: 'In queue',
+  tableStatusDisplayInitialLabel: 'Not started',
```

Comment:
Nit: Could we drop the "display" from these so as to not confuse them with things meant for the StatusDisplay component?

Reply:
Yeah actually looking at this we really don'r need "table" in them since these are just the statuses. For example:
```
statusInProgressLabel: string;
```

Reply:
Actually really confused here. we already have table column headers and status display labels, why do we need these?